GettingStartedwithAnsibleusingGNS3.md

```diff
@@ -0,0 +1,283 @@
+4. Drag the docker container to the workspace and GNS3 will automatically pull the required files from its repository over the Internet
+
+> By default, the Network Automation docker container includes Python 2.7, NAPALM, pyntc, Netmiko and Ansible. So no need to install those separately.
+
+## Setting up a Test Topology
+
+Prepare the topology as show in the below image in GNS3 Workspace:
+> NOTE: **Do not** Power-on the topology at this point
+
+![Topology](http://i68.tinypic.com/5v56ch.png "Exos_Topology")
+
+### EXOS Device Setup
+
+Power-up any **one** of the EXOSVM device from the above topology and perform the below configuration.
+
+1. Configure user-defined IP address for Default VLAN interface
+```sh
+EXOS-VM # unconfigure vlan Default ipaddress
+EXOS-VM # configure vlan Default ipaddress 192.168.122.100/24
+```
+
+2. Create an admin user and enable SSH
+
+```sh
+EXOS-VM # create account admin xtrm_user xtrm_pass
+EXOS-VM # enable SSH2
+```
+
+### Network Automation Container Setup
+
+1. Before Powering-on the container, right click on it and select "Edit config"
+2. Enable DHCP by removing the hash '#' before the DHCP config items
+3. Below is the sample config file after removing the hash:
+
+```sh
+#
+# This is a sample network config uncomment lines to configure the network
+#
+
+
+# Static config for eth0
+#auto eth0
+#iface eth0 inet static
+#	address 192.168.0.2
+#	netmask 255.255.255.0
+#	gateway 192.168.0.1
+#	up echo nameserver 192.168.0.1 > /etc/resolv.conf
+
+# DHCP config for eth0
+auto eth0
+iface eth0 inet dhcp
+```
+4. Power-on the container and launch its console
+5. Once powered-on, the container should automatically get an IP address from the NAT cloud via DHCP with default subnet of 192.168.122.0/24
+>NOTE: This is the reason why we have assigned an IP address from 192.168.122.0/24 subnet to the EXOSVM.
+6. Ping the EXOSVM and make sure that it is reachable from the Automation container
+7. Run command ```ansible --version``` to check the currently installed version of Ansible
+
+## System Setup
+
+Ideally, there is no need to install ansible on Network Automation Docker Container as it already comes pre-installed with it. But for those who are using their own standalone Linux Machines can go though the below section to install the same.
+
+### Installing Ansible
+
+1. Run the below commands to install the required files/packages:
+
+```sh
+root@NetworkAutomation-1:~# apt-get update
+root@NetworkAutomation-1:~# apt-get install git
+root@NetworkAutomation-1:~# git clone https://github.com/ansible/ansible.git --recursive
+```
+2. Do a 'ls' to make sure that ansible folder is successfully cloned. Sample command and its output is below:
+
+```sh
+root@NetworkAutomation-1:~# ls
+ansible
+```
+
+3. One can periodically pull the latest developments from Ansible's Github Repository by running the below commands:
+
+```sh
+root@NetworkAutomation-1:~# cd ansible
+root@NetworkAutomation-1:~#git pull
```

Comment:
should be space between `#` and `git pull` in that line.

Reply:
done